### PR TITLE
Add totp tilecache token support

### PR DESCRIPTION
**Pull request by Firefishy**

### CODE REVIEW

File: cookbooks/tilecache/recipes/default.rb [hunk 1]
```diff
@@ -33,9 +33,14 @@
 package "xz-utils"
 package "openssl"
 
+# oathtool for QoS token
+package "oathtool"
+
 tilecaches = search(:node, "roles:tilecache").sort_by { |n| n[:hostname] }
 tilerenders = search(:node, "roles:tile").sort_by { |n| n[:hostname] }
 
+tilecache_tokens = data_bag_item("tilecache", "tokens")
```
Comment (tomhughes) [inline cookbooks/tilecache/recipes/default.rb]: This should be in the web bag I think as the token isn't tilecache specific. If you're ready to go live with this then I'll merge the rails side and do the chef bits to configure the key.

Comment (Firefishy) [inline cookbooks/tilecache/recipes/default.rb]: Makes sense.

Comment (tomhughes) [inline cookbooks/tilecache/recipes/default.rb]: It's in `data_bag_item("web", "passwords")` as `totp_key` now.

**Comment by Firefishy** [comment]
Merged.

**Pull request closed without merging**